BoardConfigCommon.mk

```diff
@@ -117,17 +116,20 @@ endif
 
 # Encryption
 TARGET_HW_DISK_ENCRYPTION := true
+TARGET_USES_METADATA_AS_FDE_KEY := true
 
 # FM Radio
 BOARD_HAVE_QCOM_FM := true
+BOARD_HAVE_FM_RADIO := true
+TARGET_QCOM_NO_FM_FIRMWARE := true
 
 # GPS
 TARGET_NO_RPC := true
 USE_DEVICE_SPECIFIC_GPS := true
 BOARD_VENDOR_QCOM_GPS_LOC_API_HARDWARE := $(TARGET_BOARD_PLATFORM)
 
 # Kernel
-BOARD_KERNEL_CMDLINE := console=ttyHSL0,115200,n8 androidboot.console=ttyHSL0 androidboot.hardware=qcom user_debug=30 msm_rtb.filter=0x237
+BOARD_KERNEL_CMDLINE := console=ttyHSL0,115200,n8 androidboot.console=ttyHSL0 androidboot.hardware=qcom user_debug=30 msm_rtb.filter=0x237  androidboot.usbconfigfs=false
```

Comment:
additional space, probably it don't change anything

Reply:
done, corrected